cypress/integration/picking/pick_HUs_and_create_shipment.js

```diff
@@ -46,22 +46,28 @@ describe('Create test data', function() {
   });
 
   it('Create first single-HU inventory doc', function() {
-    Builder.createHUWithStock(productName, productQty, locatorId).then(huVal => (huValue1 = huVal));
+    cy.fixture('picking/pick_HUs_and_create_shipment.json').then(f => {
+      Builder.createHUWithStock(f['productName'], f['productQty'], f['locatorId']).then(huVal => (huValue1 = huVal));
```

Comment:
note that this is the correct way to make sure parameters are properly set. Before they were set with a let at the top and then assigned. That is no guarantee it will work by defining a variable on top and then assign the response of the promise. For all those within the other functions I got undefined !!!  Note that in here we deal with asyn calls and we get the correct values in cypress only after .then( ....) as I set it now.
Read doc of cypress pls.